content/doc/upgrade-guide/2.7.adoc

```diff
@@ -0,0 +1,83 @@
+
+For automatic deployments setting up a preconfigured environment e.g. using https://wiki.jenkins-ci.org/display/JENKINS/Post-initialization+script[+init.groovy+ or +init.groovy.d/+], the System property https://wiki.jenkins-ci.org/display/JENKINS/Features+controlled+by+system+properties[+jenkins.install.runSetupWizard+] allows skipping the setup wizard:
+
+ java -Djenkins.install.runSetupWizard=false -jar jenkins.war
+
+
+==== Servlet 3.1 requirement
+
+Jenkins now requires Servlet API 3.1.
+
+Administrators running Jenkins on non-default servlet containers, such as Tomcat or Websphere, need to ensure the container is capable of Servlet 3.1.
+
+The embedded Winstone-Jetty container has been upgraded to support Servlet 3.1.
+
+
+==== AJP support removed from Winstone-Jetty
+
+The embedded Winstone-Jetty container has been upgraded and dropped support for the AJP protocol. If you're using this protocol for communication between Jenkins and a reverse proxy, you will need to switch to HTTP, or use a different servlet container.
+
+If you installed Jenkins using one of the native packages, it may enable AJP support by default, even if you've never used it. As the related command-line argument to +java -jar jenkins.war+ has been dropped, it is possible that Jenkins will refuse to start after an upgrade, citing an unexpected argument. In that case, you need to edit the service configuration files (e.g. +/etc/default/jenkins+ or +/etc/sysconfig/jenkins+) to remove that parameter.
+
+
+==== Different bundled plugins mechanism
+
+In Jenkins 2, we removed the concept of _bundled plugins_. Now, Jenkins can be used with no plugins installed, and all installed plugins can be uninstalled. The new setup wizard allows users to quickly select what plugins to install from among the most popular plugins.
+
+Detached plugins, i.e. those whose functionality used to be part of Jenkins core but was moved into a plugin, will now only be installed if Jenkins detects an upgrade from a version before the plugin was detached. This is still required to retain configuration-level compatibility when upgrading Jenkins.
+
+
+==== Groovy upgraded to 2.4.7
+
+https://issues.jenkins-ci.org/browse/JENKINS-21249[JENKINS-21249], https://issues.jenkins-ci.org/browse/JENKINS-38503[JENKINS-33358]
+
+Groovy has been upgraded from 1.8.9 to 2.4.7.
+
+One possible side effect is a memory leak described in JENKINS-33358 and https://issues.apache.org/jira/browse/GROOVY-7591[GROOVY-7591]. If you experience this issue, you can set the system property +groovy.use.classvalue+ to +true+ to restore the previous behavior. Example:
+
+ java -Dgroovy.use.classvalue=true -jar jenkins.war
+
+
+==== Changes in UI theme
+
+Jenkins 2 introduced changes to the UI theme that may impact custom CSS/JS relying on the DOM or other features of the Jenkins UI.
```

Comment:
Impact needs clarification. Users are not CSS/JS/whateverUnstableWebUIStuff experts. https://github.com/oleg-nenashev/jenkins.io/blob/8f8d21aba1c340b21adfe9a83afdd7e3f9ea3ab6/content/blog/2016/2016-08-03-jenkins27x-upgrade.adoc#change-of-jenkins-html-layouts